src/game/Guards.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Guards.hp
```

Comment:
```suggestion
 * @file Guards.hpp
```
Finde ich sehr gut, dass du da Doxygen Header nutzt!